pkg/hintrunner/zero/README.md

```diff
@@ -49,7 +49,7 @@ func newAssert250bitsHint(low, high, value hinter.ResOperander) hinter.Hinter {
 5- This is the part that gets insteresting, because implementing the hint means you have to write a code in go that has the same behavior as the code of the hint in python. The difficulty of course depends on the hint. Some things that are good to know here:
 
 - It might be the case that the hint is already implemented as part of the **core** pkg (Cairo 1 hints), so you might want to check first if it is. Such is the case for `AssertLeFelt` hint. In those cases you would only need to return the corresponding hint structure from the **core** pkg.
-- The operanders allow you to get the address in memory of the variable you need to use. When this variable is a felt just reading the value from that address is enough, but sometimes the variable is actually a struct so you need to get several consecutive values in memory, for that you can use `GetConsecutiveValues(vm *VM.VirtualMachine, ref ResOperander, size int16)` method. For example in `uint256add` hint you need to add `a` and `b` which are both `Uint256` structs, so we used `GetUint256AsFelts` utility which basically calls that function with size 2.
+- The operanders allow you to get the address in memory of the variable you need to use. When this variable is a felt just reading the value from that address is enough, but sometimes the variable is actually a struct so you need to get several consecutive values in memory, for that you can use `GetConsecutiveMemoryValues(addr MemoryAddress, size int16)` method. For example in `uint256add` hint you need to add `a` and `b` which are both `Uint256` structs, so we used `GetUint256AsFelts` utility which basically calls that function with size 2.
```

Comment:
Actually, `GetUint256AsFelts` indeed uses `ReadFromAddress` 2 times to read the low and high parts of the uint256 in memory. `GetConsecutiveMemoryValues` also uses `ReadFromAddress` , inside a loop.

You should rework this section, explaining that they both use `ReadFromAddress`.
`GetUint256AsFelts` doesn't use `GetConsecutiveMemoryValues`  so the current version is wrong

Reply:
Okay, changes addressed.